.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: ruby
 rvm:
-  - 2.2
-  - 2.0
-  - 2.1
-  - 1.9
+- 2.2
+- 2.1
+- 2.0
+- 1.9
 script:
-  - rake rubocop
-  - rake test
+- rake rubocop
+- rake test
+- rake test:regression
+before_install:
+- openssl aes-256-cbc -K $encrypted_629ec55f39b2_key -iv $encrypted_629ec55f39b2_iv
```

Comment:
Key and iv won't be available for pull requests coming from forks, so you should run the regression tests only for merges. That's why in [gcloud-node](https://github.com/GoogleCloudPlatform/gcloud-node/blob/master/scripts/build.sh#L23) and [gcloud-python](https://github.com/GoogleCloudPlatform/gcloud-python/blob/master/scripts/merge.sh#L22) we have custom build scripts that are called from .travis.yml.